src/sonic-daemon-base/sonic_daemon_base/daemon_base.py

```diff
@@ -27,7 +27,7 @@
 
 # Port config information
 PORT_CONFIG = 'port_config.ini'
-PORTMAP = 'portmap.ini'
+PLATFORM_CAP = 'platform.json'
```

Comment:
In this PR and other related PRs, you name this constant, `PLATFORM_JSON`. I would prefer if you used that everywhere, and you can change it everywhere in the future if desired.

Reply:
#done